rosco/controller/src/Controllers.f90

```diff
@@ -139,7 +134,20 @@ SUBROUTINE PitchControl(avrSWAP, CntrPar, LocalVar, objInst, DebugVar, ErrVar)
         IF (CntrPar%AWC_Mode > 0) THEN
             CALL ActiveWakeControl(CntrPar, LocalVar, DebugVar)
         ENDIF
-
+             
+        ! Shutdown
+        IF (LocalVar%SD_Trigger == 0) THEN
+            LocalVar%PitCom_SD = LocalVar%PitCom
+        ELSE
+            IF (CntrPar%SD_Method == 1) THEN    !Only SD_Method==1 supported for now 
```

Comment:
A comment could help here, like "Pitch to feather at SD_MaxPitchRate".

Note that in some instances (like a downwind rotor), we may want to pitch to a stall angle.

Reply:
Resolved in 96f1f545